api/pom.xml

```diff
@@ -13,7 +13,7 @@
    <parent>
       <groupId>org.openmrs</groupId>
       <artifactId>openmrs</artifactId>
-      <version>2.2.0-SNAPSHOT</version>
+      <version>2.2.1-SNAPSHOT</version>
```

Comment:
This version will conflict with master, try something unique like 2.2.0-passwordreset please